src/chains/ethereum/transaction/tests/index.test.ts

```diff
@@ -144,6 +150,12 @@ describe("@ganache/ethereum-transaction", async () => {
         );
         assert.strictEqual(txFromRpc.type.toString(), "0x0");
       });
+      it("assumes legacy transaction if type unknown", () => {
+        txFromRpc = <LegacyTransaction>(
+          TransactionFactory.fromRpc(unknownTypeTx, common)
+        );
+        assert.strictEqual(txFromRpc.type.toString(), "0x0");
```

Comment:
Can you also add a test to handle the case where the unknown `type` is `"0xc0"` and one more where it some value greater than `"0xc0"`?